EpikCord/managers/channel_manager.py

```diff
@@ -30,5 +30,5 @@ async def add(self,guild_id,data:dict):
         channel_data = await self.client.http.post(f"guilds/{guild_id}/channels", data=data)
         self.channels.append(channel_data)
         
-    async def edit_pos(self,guild_id,data:dict):
+    async def edit_position(self,guild_id,data:dict):
```

Comment:
Should be part of `GuildChannel` and should just take in an `int` for the position and not use that endpoint but `/guilds/channel/ID/` iirc

Reply:
ok will do that

Reply:
according to discord API, the endpoint u gave is nopt correct

Reply:
Send a PATCH to `/channels/ID`, I think I already covered this though.